hackathons.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
```

Comment:
Its usually better to add new items to the end not the beginning, but not the end of the world